image_bootstrap/patches/cloud-init-0-7-6-pkgbuild.patch

```diff
@@ -33,6 +33,9 @@
 +    sed -i '1s/python$/&2/' $pkgname-$PKGVER/tools/read*
 +    sed -i 's:/etc/systemd:/usr/lib/systemd:g' $pkgname-$PKGVER/setup.py
 +    sed -i 's,name: ubuntu,name: archlinux,;s,gecos: Ubuntu,gecos: Archlinux,' $srcdir/$pkgname-$PKGVER/config/cloud.cfg
++
++    cd ${srcdir}/${pkgname}-${PKGVER}
++    patch -Np1 -i "${OLDPWD}"/../cloud-init-0-7-6-uid.patch
```

Comment:
I considered that any only did not put it in source to save the hardcoded SHA1, because it doesn't add protection for us. I can add it, if prefer.

I wonder if `"${srcdir}/cloud-init-0-7-6-uid.patch" works. My impression was that `${srcdir}` is for patches shipped by upstream; I have seen `../foo-patch` in practice, e.g. at https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/cloud-init. For that location, `${srcdir}` does not seem to work.